packages/java-shell/src/test/resources/collection/insertOne.expected.txt

```diff
@@ -1,3 +1,3 @@
 { "acknowledged": true, "insertedId": "UNKNOWN" }
 true
-[ { "_id": <ObjectID>, "a": 1, "objectId": <ObjectID>, "maxKey": {"$maxKey": 1}, "minKey": {"$minKey": 1}, "binData": {"$binary": {"base64": "MTIzNA==", "subType": "10"}}, "date": {"$date": {"$numberLong": "1355875200000"}}, "isoDate": {"$date": {"$numberLong": "1355875200000"}}, "numberInt": 24, "timestamp": {"$timestamp": {"t": 100, "i": 0}}, "undefined": null, "null": null, "uuid": <UUID> } ]
+[ { "_id": <ObjectID>, "a": 1, "objectId": <ObjectID>, "maxKey": {"$maxKey": 1}, "minKey": {"$minKey": 1}, "binData": {"$binary": {"base64": "MTIzNA==", "subType": "10"}}, "date": {"$date": {"$numberLong": "1355875200000"}}, "isoDate": {"$date": {"$numberLong": "1355875200000"}}, "numberInt": 24, "timestamp": Timestamp{value=429496729600, seconds=100, inc=0}, "undefined": null, "null": null, "uuid": <UUID> } ]
```

Comment:
This is a bit curious – isn’t the output supposed to be EJSON (just basing off `$maxKey`, `$minKey`, `$binary` etc.)?

Reply:
Actually yes, it should.
I can fix it but then I'll need to find a better way to check if values were converted to JS or not.
I'll do it but later